models/library_item.py

```diff
@@ -1,20 +1,24 @@
-from sqlalchemy import Column, Integer, Numeric
-
-import utils.db
+from resources import db
 from utils.helpers import percentage_from
 
 DAILY_RENT_PRICE_PERCENT = 25
 
 
-class LibraryItem(utils.db.Base):
+class LibraryItem(db.Model):
     __tablename__ = "library_item"
-    id = Column(Integer, primary_key=True)
-    pledgePrice = Column(Numeric)
-    quantity = Column(Integer)
+    id = db.Column(db.Integer, primary_key=True)
+    pledge_price = db.Column(db.Numeric)
+    quantity = db.Column(db.Integer)
 
     def __repr__(self):
         return str(self.__dict__)
 
+    @staticmethod
```

Comment:
Better to use .get("key", None) instead of ["key"], because ["key"] can raise KeyError